ansible/roles/postgresql/tasks/main.yml

```diff
@@ -89,15 +89,6 @@
   tags: postgresql_conf
   notify: Reload postgres
 
-- name: PostgreSQL recovery configuration
-  template:
-    src: "recovery.conf.j2"
-    dest: "{{ postgresql_home }}/recovery.conf"
-    owner: "postgres"
-    group: "postgres"
-  notify: Restart postgres
-  when: is_pg_standby
-
```

Comment:
removed this from there since it prevents PG setup. Once the file is created it doesn't need to be managed by ansible I don't think